### DISCUSSION TITLE
resolve swapping of symmetric nodes

### DISCUSSION THREAD

**Discussion by ddinghooo** [talmolab/sleap#1670] [Help!]
Hello, I trained 580 frames of image data in a bottom-up method for multi-animal tracking. In addition, Max stride was set to 32 so that all body parts of the mouse could fit within the receptive field. 
![image](https://github.com/talmolab/sleap/assets/157553047/a3cc4eb1-5a4b-4eb5-a153-70bda5591f79)

However, when tracking videos of mouse movements after learning, I could observe that the left ear and the right ear changed very often. So I want to know how to solve this problem.
![image](https://github.com/talmolab/sleap/assets/157553047/2228366b-11b6-4bb1-8d09-b39b529ddba3)

I checked #1368 to resolve the issue. I think the size of the receptive field is sufficient because Max stride has already been trained to 32. Also I checked the labeled data to see if there was a frame labeled with the left ear and the right ear in reverse. So I'm going to change rotation augmentation to +-180 and re-learn.

Or could it help to change the skeleton settings? I want to connect neck and leftear, and not connect with rightear.

I would appreciate it if you could let me know if there's a good way!

**Comment by Marielamend** [comment]
Hello @ddinghooo,


That's great that you were able to get some insight from that discussion! Changing augmentation and re-checking to see your labeled data will help! 


In terms of your skeleton, I would suggest connecting the neck node to the left ear and the right ear (making the neck node the source) because the neck node tends to be located more easily than the nose hence making it easier to locate the ears if they are the destination of that neck node. 

Here are a few resources that can help visualize this!


[Guide](https://sleap.ai/guides/skeletons.html): Tips on skeleton design.
[Datasets](https://sleap.ai/datasets.html): See the mouse datasets we provide for reference on skeletons in different conditions.
You can also take a look at an example template under Skeleton.
![image](https://github.com/talmolab/sleap/assets/120222245/9def9935-71e1-40f6-9c63-bbe606986f8d)


Have a great day,
Mariela
